src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
```

Comment:
`fd-list__item--action` is used together with `fd-list__item` and `fd-list__item ` already had `display:flex` so you don't need to set it again.

Reply:
Changes implemented